templates/ifb/user/add_user.sh

```diff
@@ -20,7 +20,7 @@ touch "{{ user.home }}/.ssh/authorized_keys"
 echo 'Host *' > "{{ user.home }}/.ssh/config"
 echo '  StrictHostKeyChecking no' >> "{{ user.home }}/.ssh/config"
 echo '  UserKnownHostsFile=/dev/null' >> "{{ user.home }}/.ssh/config"
-chown -R {{ user.uidnumber }}:{{ user.gidnumber }} "{{ user.home }}"
+chown -R {{ user.uid }}:{{ user.uid }} "{{ user.home }}"
```

Comment:
I usually prefer and advise to use ids rather than names.
If scripts were to be executed in a container or different server not connected to ldap, using name fail while id always work.

Reply:
yes I know we already have this conversation ;)